internal/pkg/diff/diff.go

```diff
@@ -164,6 +155,48 @@ func (d *Differ) doDiff(state model.ObjectState) (*Result, error) {
 	return result, nil
 }
 
+func (d *Differ) diffValues(remoteState, remoteValue, localState, localValue interface{}) string {
+	d.reporter = newReporter(remoteState, localState)
+	cmp.Diff(remoteValue, localValue, d.newOptions())
+	return d.reporter.String()
+}
+
+func (d *Differ) newOptions() cmp.Options {
+	return cmp.Options{
+		// Compare Config/ConfigRow configuration content ("orderedmap" type) as map (keys order doesn't matter)
+		cmp.Transformer("orderedmap", utils.OrderedMapToMap),
+		// Compare relations in human-readable form
+		cmp.Transformer("relations", func(relations model.Relations) string {
+			var out []string
+			for _, relation := range relations {
+				out = append(out, d.relationToString(relation))
+			}
+			sort.Strings(out)
+			return strings.Join(out, "\n")
+		}),
+		// Compare strings by lines
+		cmpopts.AcyclicTransformer("strByLine", func(s string) []string {
+			return strings.Split(s, "\n")
+		}),
+		// Custom reporter
+		cmp.Reporter(&d.reporter),
+	}
+}
+
+func (d *Differ) relationToString(relation model.Relation) string {
+	otherSideDesc := ``
+	if relationParent := d.reporter.ParentKey(); relationParent != nil {
+		otherSideKey := relation.OtherSideKey(relationParent)
+		if otherSide, found := d.state.Get(otherSideKey); found {
+			otherSideDesc = `"` + otherSide.Path() + `"`
+		}
+		if len(otherSideDesc) == 0 {
+			otherSideDesc = otherSideKey.Desc()
+		}
+	}
+	return relation.Desc() + ` ` + otherSideDesc
```

Comment:
napr. `variables from "main/extractor/ex-generic-v2/with-variables-1/variables"`

Reply:
ak by sme nevedeli ziskat `path`, tak sa vypise `key desc`, ... ak ani to, tak vystup bude prazdny (to by nemalo nikdy nastat).